pkgs/development/python-modules/progressbar2/default.nix

```diff
@@ -25,7 +25,7 @@ buildPythonPackage rec {
 
   postPatch = ''
     rm -r tests/__pycache__
-    rm tests/*.pyc
+    find tests -name "*.pyc" | xargs -r rm
   '';
```

Comment:
these lines aren't needed anymore, can you just remove postPatch altogether?

Reply:
I don't recall what this was for now unfortunately (doh). But, if this could be removed, why would the build have failed and this have fixed it for @Pamplemousse ?

Reply:
Just FYI

It failed because `tests/*.pyc` does not expand to anything, hence `rm` ends up with an empty operand.

My fix works because of the `-r` passed to `xargs`; It would have been more explicit if I used its expanded version `--no-run-if-empty`... (cf. https://linux.die.net/man/1/xargs)